### DISCUSSION TITLE
Using custom fonts with fabric in NodeJS with V6

### DISCUSSION THREAD

**Discussion by maxswjeon** [fabricjs/fabric.js#8979] [Q&A] [answered]
Hi, I'm using Fabric V6 and it's amazing!

I want to register a font, found #7065, but I cannot access nodeCanvas on V6.0

How did it changed?

**Comment by ShaMan123** [comment, marked as answer]
You can import node canvas yourself, there is no fabric namespace anymore.
V6 rocks, doesn't it? I can't wait for v7
